Header Header
bool poi_present
float64 poi_depth
float64 poi_angle
bool poi_deviation
bool obstacle_present
string obstacle_label
string turn_direction
float64 forward_distance